tardis/resources/dronestates.py

```diff
@@ -19,13 +20,14 @@
 
 
 async def batchsystem_machine_status(state_transition, drone: "Drone",
```

Comment:
``state_transition`` is some ``Dict[State, State]``?

```suggestion
async def batchsystem_machine_status(state_transition: Dict[State, Type[State]], drone: "Drone",
```